landscape/client/manager/aptsources.py

```diff
@@ -132,7 +132,13 @@ def _handle_sources(self, ignored, sources):
             if os.path.isfile(saved_sources):
                 shutil.move(saved_sources, self.SOURCES_LIST)
 
-        if self.registry.config.manage_sources_list_d:
+        if self.registry.config.manage_sources_list_d not in (
```

Comment:
It would be nice if there was a helper function for this, so it could be used in other config parameters

Reply:
I'm hesitant to over-engineer this fix, as I hope we can replace our dated config parsing with the more modern approach, which should handle datatypes like this internally.

Reply:
What do you think about just a variable in landscape/constants.py like FALSE_VALUES = ["false", "0", "no", ..]

Reply:
Hmm yeah that's fine.